lib/validation/rules/email.ts

```diff
@@ -1,8 +1,14 @@
-import { email as baseEmail, helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
+import { email as baseEmail } from '../validators'
+
+export const message = {
+  en: 'The Email is invalid.',
+  ja: 'メールアドレスが正しくありません。'
```

Comment:
```suggestion
  ja: 'Emailの形式が正しくありません。'
```

We use the word "Email" instead of "メールアドレス" in almost everywhere.

Reply:
Fixed: e5c53aa